solidity/contracts/Chainlinked.sol

```diff
@@ -31,14 +31,14 @@ contract Chainlinked {
     returns(bytes32)
   {
     requests += 1;
-    _run.externalId = keccak256(this, requests);
+    _run.requestId = keccak256(this, requests);
```

Comment:
I know this code predates the PR, but could we just use requests as the requestId? What do we gain from hashing it?

Reply:
Sure, I think we can just use the uint instead of a hash. It'll be cheaper on gas so I'll make that change.

Reply:
It does mean that the oracle function id might change:

```
  bytes4 internal constant oracleFid = bytes4(keccak256("requestData(uint256,bytes32,address,bytes4,bytes32,bytes)"));
```

Change that last bytes32 to uint256, unless we coerce it to bytes32 which I think is overkill.